torch/csrc/jit/autodiff.cpp

```diff
@@ -99,7 +99,7 @@ static std::vector<Value*> gradientForNode(Node* node, ArrayRef<Value*> grad_val
       return {grads.at(0) * inputs.at(1), grads.at(0) * inputs.at(0)};
 
     } else if (node->matches("aten::sigmoid(Tensor self) -> Tensor")) {
-      return {grads.at(0) * outputs.at(0) * (1 - outputs.at(0))};
+      return {(1 - outputs.at(0)) * outputs.at(0) * grads.at(0)};
```

Comment:
Please add a comment/TODO here that the order of operations matters for different architectures and to look into it

Reply:
@zou3519 thanks for checking.  I added a TODO comment.